templates/projects/js/annotation_guidelines_button.js

```diff
@@ -19,5 +19,5 @@ function setup_task_timeout_display(timeout, original_timeout) {
             });
         },  timeout * 1000);
 }
-setup_task_timeout_display({{ project.timeout or 0 }}, {{ project.original_timeout or project.timeout or 0 }});
+//setup_task_timeout_display({{ project.timeout or 0 }}, {{ project.original_timeout or project.timeout or 0 }});
```

Comment:
We could change this to be per-project, instead of for all projects.

This could be done by uncommenting the line to keep it, and instead, assign a global variable to the timeout method. A project owner may then cancel the timer themselves, if they choose to.

```js
var taskTimeOutInterval;

function setup_task_timeout_display(timeout, original_timeout) {
  if (timeout && parseInt(timeout)) {
    taskTimeOutInterval = setTimeout(function() {
      // ...
    },  timeout * 1000);
}
```

To disable the task timeout feature on a project, a project owner may add the following into the task presenter code:

```js
clearInterval(taskTimeOutInterval);
```